build_deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+IMAGE="${IMAGE:-quay.io/cloudservices/provisioning-backend}"
+IMAGE_TAG="$(git rev-parse --short=7 HEAD)"
+SMOKE_TEST_TAG="latest"
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
+    echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN  must be set"
+    exit 1
+fi
+
+# Login to quay
+podman login \
+    -u DOCKER_LOGIN_USER="$QUAY_USER" \
+    -p DOCKER_LOGIN_TOKEN="$QUAY_TOKEN" \
+    quay.io
+
+# Login to registry.redhat
+podman login \
+    -u DOCKER_LOGIN_USER="$RH_REGISTRY_USER" \
+    -p DOCKER_LOGIN_TOKEN="$RH_REGISTRY_TOKEN" \
+    registry.redhat.io
+
+# build and push
+make build-podman \
+    DOCKER_BUILD_OPTS=--no-cache \
+    # TODO opts for image base/tag 
+
+# push to logged in registries and tag for SHA
+docker tag "${IMAGE}:${IMAGE_TAG}" "${IMAGE}:${SMOKE_TEST_TAG}"
+docker push "${IMAGE}:${SMOKE_TEST_TAG}"
```

Comment:
can we use podman here? I'd stick with one tool and podman is bit lighter dependency, so we wanted to use podman if at all possible.

Reply:
Absolutely - thanks for the quick review! Will make all this consistent.